src/lib/progress-bar/progress-bar.ts

```diff
@@ -27,6 +35,11 @@ export class MatProgressBarBase {
   constructor(public _elementRef: ElementRef) { }
 }
 
+/** Last animation end data. */
+export interface AnimationEndData {
```

Comment:
Ah, one last thing I missed earlier- this should be `ProgressAnimationEnd`; having `Data` at the end is inconsistent with other events around the library

Reply:
Done.